### PR TITLE
feat: Adding Chatbot to website

### PR DESCRIPTION
**Pull request by Ojas-Arora**
## Related Issue

fixes: #399 

## Description

@Sanchitbajaj02 I have added the chatbot to the website. Kindly review this.

## Screenshots

image

## Summary by CodeRabbit

- **New Features**
	- Integrated an external script for enhanced functionality on the home page.
	- Improved text clarity and readability in descriptions for design professionals and developers.

### CODE REVIEW

File: src/components/pages/home/index.tsx [hunk 1]
```diff
@@ -256,6 +256,7 @@ function HomePage({ accountId }: { accountId: string | undefined }) {
         <section className="w-full">
               <Newsletter />
         </section> 
+        <script src="//code.tidio.co/fdylvmddtyb7vzsk5frdt3ncrk6cwobs.js" async></script>
```
Comment (Sanchitbajaj02) [inline src/components/pages/home/index.tsx:259]: Why is it a script tag? Please use the library to set it up as the CDN is not acceptable in large scale projects. Also you are exposing your tidio ID

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by Ojas-Arora** [comment]
@Sanchitbajaj02 Ok I will do so and for Tidio to work Id needs to be exposed

**Comment by Ojas-Arora** [comment]
@Sanchitbajaj02 I have updated the branch and without exposing the Id the chatbot will not work.

**Review by Sanchitbajaj02 — APPROVED** [review]
Approving but it is not satisfactory output. Will do full review and unit testing on it

**Comment by Ojas-Arora** [comment]
@Sanchitbajaj02 Sir u have approved the changes but I haven't received any points for this. Kindly see what to do for this.

**Comment by Sanchitbajaj02** [comment]
> @Sanchitbajaj02 Sir u have approved the changes but I haven't received any points for this. Kindly see what to do for this.

Fix the conflicts to get it merged

**Comment by Ojas-Arora** [comment]
@Sanchitbajaj02 done

**Comment by Ojas-Arora** [comment]
@Sanchitbajaj02 Kindly review this as tomorrow is last day.

**Pull request merged**